MessageViewController/MessageViewController.swift

```diff
@@ -71,6 +71,8 @@ open class MessageViewController: UIViewController, MessageAutocompleteControlle
         }
     }
 
+    open func didLayout() { }
```

Comment:
Some header docs to give a hint as to what this can be used for might help :)

Reply:
PR!